cmd/keycloakb/keycloak_bridge.go

```diff
@@ -907,7 +909,8 @@ func main() {
 		managementSubroute.Path("/realms/{realm}/users/{userID}").Methods("PUT").Handler(updateUserHandler)
 		managementSubroute.Path("/realms/{realm}/users/{userID}").Methods("DELETE").Handler(deleteUserHandler)
 		managementSubroute.Path("/realms/{realm}/users/{userID}/groups").Methods("GET").Handler(getGroupsForUserHandler)
-		managementSubroute.Path("/realms/{realm}/users/{userID}/groups").Methods("PUT").Handler(setGroupsToUserHandler)
```

Comment:
setGroupsToUser is not needed anymore?

Reply:
Yes... it is replaced by Add/Delete methods